.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
what does this do? It should have been filtered out in git ignore isn't it?

Reply:
this was intentional. I have code spell checker plugin.